Bottom: ebef106b196b9ca590348afca8428a82520ed5a6
Top:    d1354e7b9d1b968ca9b674b3fd02b5d2f914ee94
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 17:22:39 +0800

Refresh of debug.patch

---

diff --git a/lib/cache.c b/lib/cache.c
index a074f23..7a2caa6 100644
--- a/lib/cache.c
+++ b/lib/cache.c
@@ -658,8 +658,6 @@ int cache_update_offset(struct mapent_cache *mc, const char *mkey, const char *k
 		return CHE_FAIL;
 
 	me = cache_lookup_distinct(mc, key);
-	if (me)
-		error(logopt, "owner %p owner->key %s me %p me->key %s", owner, owner->key, me, me->key);
 	if (me && me->age == age) {
 		if (me == owner)
 			return CHE_DUPLICATE;
